frontend/package.json

```diff
@@ -49,6 +49,7 @@
     "react": "^17.0.2",
     "react-beautiful-dnd": "^13.1.0",
     "react-csv": "^2.2.2",
+    "react-dev-inspector": "^2.0.1",
```

Comment:
The version is also bumped 🤔
Is it compatible with our react? I saw that it was using [`react@18`](https://github.com/zthxxx/react-dev-inspector/blob/dev/packages/inspector/package.json#L121)

Reply:
amended version back to 1.8.6